CONTRIB_pl.md

```diff
@@ -56,7 +56,7 @@ git version 2.17.1.windows.2
     ![pliki na dysku](docs/img/clone-fs.png)
 
     * Dodaj upstream 
-        * przejdź do orginalnego źrudła i skopiuj link z ``Clone or download``. Następnie w cmd wpisz:
+        * przejdź do orginalnego źrudła i skopiuj link z ``Clone or download`` (tak jak poprzednio). Następnie w cmd wpisz:
```

Comment:
```suggestion

```

Proponuje usunąć to zdanie. Nie jest ono potrzebne. Wystarczy wpisac to, co poniżej. Tu mamy instrukcje tego jak pracować z naszym shardem. Adres sie nie zmieni na GH. :)

PS: Źróóóóódło